config.go

```diff
@@ -19,6 +19,7 @@ type Configs struct {
 	ExpansionfilePath string          `env:"expansionfile_path"`
 	Track             string          `env:"track,required"`
 	UserFraction      float64         `env:"user_fraction,range]0.0..1.0["`
+	UpdatePriority    int             `env:"update_priority,range]0..5["`
```

Comment:
According to the docs, the range should be inclusive-inclusive instead of exclusive-exclusive